client/fossildb-client

```diff
@@ -15,8 +17,7 @@ def main():
             stub.Backup(proto.BackupRequest()),
         'restore': lambda stub:
             stub.RestoreFromBackup(proto.RestoreFromBackupRequest()),
-        'health': lambda stub:
-            stub.Health(proto.HealthRequest())
+        'health': None
```

Comment:
This doesn't make sense anymore. Maybe remove the stub argument and replace (for our stub) with a function used in backup & restore. Also health could still be part of that map. I'll prepare a commit with a proposal.